src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -101,20 +101,6 @@ case class LocalBackend(actorSystem: ActorSystem) extends Backend with SharedFil
     }
   } map CompletedExecutionHandle
 
-  def useCachedCall(cachedBackendCall: BackendCall, backendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = Future {
```

Comment:
I'll rebase this on top of yours since yours when that's merged because yours is more involved